recipe/meta.yaml

```diff
@@ -522,7 +541,11 @@ requirements:
     - sysroot_linux-aarch64 {{ cudavars[major_minor]["sysroot_version"] }}  # [aarch64]
 {% endif %}
   run_constrained:
+{% if major_version < 11 %}
     - __cuda >={{ major_minor }}
+{% else %}
+    - __cuda >={{ major_version }}.0.<{{ major_version + 1 }}.0
```

Comment:
I think this should just be `__cuda >={{major_version}}.0` as you should be able to use a CUDA Toolkit 11.x version with a driver that supports 12.x once it releases.

Reply:
Are there any cases where that is not true? Where an older toolkit won't work for a sufficiently new driver?

Reply:
> Each release of the CUDA Toolkit requires a minimum version of the CUDA driver. The CUDA driver is backward compatible, meaning that applications compiled against a particular version of the CUDA will continue to work on subsequent (later) driver releases. 

https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html

NVidia primises to support older cuda versions on newer drivers (backward compatibility). So the requested change is valid - thanks for pointing that out!